src/relay/backend/contrib/cmsisnn/target.cc

```diff
@@ -31,10 +34,11 @@ tvm::transform::Pass RelayToTIR();
 runtime::Module TIRToRuntime(IRModule mod, Target target);
 
 TVM_REGISTER_TARGET_KIND("cmsis-nn", kDLCPU)
+    .add_attr_option<Array<String>>("mattr")
+    .add_attr_option<String>("mcpu")
```

Comment:
just musing, i'm sensing we'll shortly want a way to express this as "whatever target_host did"